document_hosting/controllers/main.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+# Copyright 2015-2016 Odoo S.A.
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import http
+from odoo.fields import Date
+from odoo.http import request
+
+
+class DocumentWebsite(http.Controller):
+    @http.route("/documents/<int:oid>", auth="public", website=True)
+    def get_document(self, oid=-1):
+        """Render a http response for a document"""
+
+        if not request.website.display_document_page:
+            return request.not_found()
+
+        document_mgr = request.env["document_hosting.document"]
+        doc = document_mgr.sudo().browse(oid)
+        return (
+            request.env["ir.binary"]
+            ._get_image_stream_from(
+                doc, field_name="document", filename_field="filename"
+            )
+            .get_response()
+        )
+
+    @http.route("/documents", auth="public", website=True)
+    def template_website_document(self, date_begin=None, date_end=None, **kw):
+        """"""
```

Comment:
Remove this or privide a usefull docstring.

Reply:
Fixed ! thanks for the review